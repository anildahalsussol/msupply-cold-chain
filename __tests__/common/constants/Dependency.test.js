import { DEPENDENCY } from '~constants';

describe('Constants: DEPENDENCY', () => {
  it('Matches snapshot', () => {
    expect(DEPENDENCY).toMatchSnapshot();
  });
});
