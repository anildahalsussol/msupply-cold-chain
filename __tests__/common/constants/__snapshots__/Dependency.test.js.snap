// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Constants: DEPENDENCY Matches snapshot 1`] = `
Object {
  "ACKNOWLEDGE_BREACH_MANAGER": "acknowledgeBreachManager",
  "BLUETOOTH": "bleService",
  "BREACH_CONFIGURATION_MANAGER": "breachConfigurationManager",
  "BREACH_MANAGER": "breachManager",
  "CHART_MANAGER": "chartManager",
  "CONSECUTIVE_BREACH_MANAGER": "consecutiveBreachManager",
  "CUMULATIVE_BREACH_MANAGER": "cumulativeBreachManager",
  "DATABASE": "database",
  "DATABASE_UTILS": "databaseUtils",
  "DEVICE": "device",
  "DEV_MANAGER": "devManager",
  "DOWNLOAD_MANAGER": "downloadManager",
  "EXPORT_SERVICE": "exportService",
  "FORMAT_SERVICE": "formatService",
  "LOCATOR": "DependencyLocator",
  "LOGGER_SERVICE": "loggerService",
  "LOG_TABLE_MANAGER": "logTableManager",
  "MIGRATION": "migrationService",
  "PERMISSION_SERVICE": "permissionService",
  "REPORT_MANAGER": "reportManager",
  "SENSOR_MANAGER": "sensorManager",
  "SENSOR_STATUS_MANAGER": "sensorStatusManager",
  "SETTING_MANAGER": "settingManager",
  "SYNC_OUT_MANAGER": "syncOutManager",
  "SYNC_QUEUE_MANAGER": "syncQueueManager",
  "SYNC_SERVICE": "syncService",
  "TEMPERATURE_LOG_MANAGER": "temperatureLogManager",
  "UTIL_SERVICE": "utilService",
}
`;
